manifests/params.pp

```diff
@@ -64,5 +64,7 @@
   $pulpcore_postgresql_ssl_cert    = '/etc/pki/katello/certs/pulpcore-database.crt'
   $pulpcore_postgresql_ssl_key     = '/etc/pki/katello/private/pulpcore-database.key'
   $pulpcore_postgresql_ssl_root_ca = '/etc/pki/tls/certs/ca-bundle.crt'
+  $pulpcore_allowed_import_path    = ['/var/lib/pulp/sync_imports']
+  $pulpcore_allowed_export_path    = []
```

Comment:
I would set these to the defaults Katello expects, the values you have here -- https://github.com/theforeman/foreman-installer/pull/615/files#diff-f333bd70087fcf1988e26c39646174b3aac006d11e0412c84cbe081b0d573598R2-R3

Reply:
I had set them as per [this comment](https://github.com/theforeman/puppet-foreman_proxy_content/pull/297#discussion_r527992505) above.  should I change it back?

Reply:
I think https://github.com/theforeman/puppet-foreman_proxy_content/pull/297#discussion_r528764632 provides what I would envision. This module is focused heavily on deploying Pulp for Katello's use case. The difference is whether deploying in a mirror setup or not. 

Would a user ever need to configure these paths differently?

Reply:
I don't think so.  Updating

Reply:
@ehelms Changed back to your suggestion.  do I also need to make changes in `init.pp`?